### PR TITLE
Feat/translation

### PR DESCRIPTION
**Pull request by macintoshpie**
#### Any background context you want to provide?
BuildingSync v3 is being developed with breaking changes from v2

#### What does this PR do?
Creates the translation file for transforming breaking v3 changes from v2 bsync instance documents.
This includes translations for the breaking changes that were already merged into v3:
- #335 
- #341 

#### How should this be manually tested?
See validation_spec.rb. You can run locally and see the translated v2 file in spec/output

#### What are the relevant tickets?
#347

### CODE REVIEW

File: spec/validation_spec.rb [hunk 1]
```diff
@@ -389,3 +389,52 @@
...
+RSpec.describe 'Version translation from v2 to v3' do
+  before :all do
+    # TODO: DRY this up --- duplicate code from example validators spec above
+  
+    # Nokogiri doesn't seem to support XSDs which import other schemas with URLs
+    # for the schemaLocation. To allow testing, we download the imported schema
+    # and point schemaLocation to it instead
+    schema_doc = Nokogiri::XML(File.read('BuildingSync.xsd'))
+
+    GEOJSON_XSD_PATH = 'geojson.xsd'
```
Comment (nllong) [inline spec/validation_spec.rb:402]: Isn't this supposed to be GBXML_XSD_PATH and not GEOJSON?

File: translation/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+We will list any known limitations of the translators in the [limitations](#limitations) section below
+
+## Usage
+If using Linux or a Mac, make sure xsltproc is installed by running
+```bash
+xsltproc
+```
+Then transform your file(s) with
+```bash
+xsltproc [-o <output>] <stylesheet> <file> [<file> ...]
```
Comment (nllong) [inline translation/README.md:13]: 👍

File: translation/v2_to_v3.xsl [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  <xsl:template match="@xsi:schemaLocation">
+    <xsl:attribute name="xsi:schemaLocation">http://buildingsync.net/schemas/bedes-auc/2019 https://raw.githubusercontent.com/BuildingSync/schema/v3.0.0/BuildingSync.xsd</xsl:attribute>
+  </xsl:template>
+
+  <xsl:template match="@auc:version">
+    <xsl:attribute name="auc:version">3.0.0</xsl:attribute>
+  </xsl:template>
+
+  <!-- Map "Repair leaks / seal ducts" to "Repair leaks in ducts" -->
+  <xsl:template match="auc:MeasureName[. = 'Repair leaks / seal ducts']">
```
Comment (nllong) [inline translation/v2_to_v3.xsl:43]: noice!

**Review by nllong — CHANGES_REQUESTED** [review]
Looks sweet! Only one question that needs resolved...

File: spec/validation_spec.rb (hunk 1 above)
Comment (macintoshpie) [reply spec/validation_spec.rb:402]: whoops, not sure how that happened, I'll get that fixed

**Comment by macintoshpie** [comment]
@aviveiros11 or @JieXiong9119 can one of you also take a look at this?

File: spec/validation_spec.rb (hunk 1 above)
Comment (nllong) [reply spec/validation_spec.rb]: terminology leaking from other projects :(

**Review by aviveiros11 — APPROVED** [review]
Worked as expected for me 🚀

**Comment by JieXiong9119** [comment]
I just tested it on pc. Working fine!

**Review by JieXiong9119 — APPROVED** [review]

**Review by nllong — APPROVED** [review]
looks shweet! Thanks for fixing the GeoJSON name.

**Pull request merged**